src/robust.jl

```diff
@@ -0,0 +1,129 @@
+# Robust Statistics
+
+#############################
+#
+#   Trimming outliers
+#
+#############################
+
+# Trimmed set
+"""
+    trim(x; prop=0.0, count=0)
+
+
+Remove either `count` or proportion `prop` of the highest and lowest elements
```

Comment:
"Remove... in `x`" could be interpreted incorrectly as meaning that the original `x` is modified. Would be better to say something like "Return a copy of `x` with ... removed."